nxviz/plots.py

```diff
@@ -459,15 +473,68 @@ def compute_node_label_positions(self):
             else:  # left side
                 rot = theta_deg - 180
 
-            xs.append(x)
-            ys.append(y)
-            has.append(ha)
-            vas.append(va)
-            rotations.append(rot)
+            # Store values
+            self._store_node_label_meta(x, y, tx, ty, rot)
+
+    @staticmethod
+    def nz_sign(xy):
```

Comment:
Please add a docstring that describes what this function is intended to do. Also, the naming is a bit cryptic: what does `nz` in `nz_sign` stand for?